Zukini.UI/Zukini.UI.csproj

```diff
@@ -60,6 +60,7 @@
   <ItemGroup>
     <Compile Include="CoypuExtensions.cs" />
```

Comment:
Should CoypuExtensions.cs be removed now?

Reply:
see previous comment, I think it still has a place for the Table extensions